src/core/tFunctions.ml

```diff
@@ -210,6 +210,9 @@ let rec is_parent csup c =
 		| None -> false
 		| Some (c,_) -> is_parent csup c
 
+(* for convenience *)
+let extends c csup = is_parent csup c
```

Comment:
This seems superfluous.

Reply:
I was looking for a method to check if `c` extends `csup` for a few minutes. And then even started to write one. And only then I realized it could be "is_parent" instead of "extend" :)

Reply:
Well, I actually prefer `extends` because you can infix-read it as `c extends csup`. But we shouldn't have both methods with different argument order.

Reply:
Ok, I'll refactor `is_parent` in a separate commit to development.